src/org/zaproxy/zap/extension/ascanrulesAlpha/ZapAddOn.xml

```diff
@@ -34,10 +34,12 @@
 		<ascanrule>org.zaproxy.zap.extension.ascanrulesAlpha.TestUserAgent</ascanrule>

```

Comment:
Should also have a new entry in the `changes` section, something like: `Added Hidden Files Scanner (issue 4585)`